main.go

```diff
@@ -213,6 +229,14 @@ func main() {
 		failf("Error running command %s: %s", config.Command, err)
 	}
 
+	// Only cache if npm command is install, node_modules could be included in the repository
+	if (len(args) != 0) && (args[0] == "install") {
```

Comment:
`args[0] == "install"` is a very brave hardcode. please make it more dynamic and keep in mind that all the following are completely valid:
```
npm i ...
npm install ...
npm add ...
(maybe more?)
```